src/runtime_src/core/pcie/linux/scan.cpp

```diff
@@ -1149,6 +1149,37 @@ get_uuids(std::shared_ptr<char>& dtbbuf, std::vector<std::string>& uuids)
   return uuids.size() ? 0 : -EINVAL;
 }
 
+/*
+ * This is for the RHEL 8.x kernel. From the RHEL 8.x kernel removed the runtime_active_kids sysfs
+ * node for the Linux power driver. Hence, to get the active kids under a bridge we need this
+ * alternative solution.
+ */
+int
+get_runtime_active_kids(std::string &pci_bridge_path)
+{
+  int curr_act_dev = 0;
+  std::vector<bfs::path> vec{bfs::directory_iterator(pci_bridge_path), bfs::directory_iterator()};
+
+  // Check number of Xilinx devices under this bridge. 
+  for (auto& path : vec) {
+    if (bfs::is_directory(path)) {
```

Comment:
Use early exit from the loop by inverting this check

Reply:
Modified. Thanks.